src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/Converters.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.companieshouse.overseasentitiesapi.utils;
+
+import java.time.LocalDate;
+import uk.gov.companieshouse.api.model.utils.AddressApi;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.AddressDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.Address;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.PersonName;
+
+public class Converters {
+
+  public static Address addressDtotoAddress(AddressDto value) {
```

Comment:
we must be doing this elsewhere already? Can reuse?

Reply:
So Barry ended up making something similar in AdressUtils but that's the only one I'm familiar with and that's also in PR

Personally, I think it might be best to keep all the Dto -> models in one place and there are others that are needed in here 

Either way though I'm good